### ISSUE TITLE
Scroll bug

### ISSUE THREAD

**Issue by OctopBP** [tabler/tabler-icons#652]
I am experiencing a scroll bug on https://tabler-icons.io, which makes it difficult to find anything.

https://user-images.githubusercontent.com/17803104/236582003-66e320e7-c6ef-48e2-ab02-c9c54baf79e7.mp4

**Comment by codecalm** [comment]
yes, i know it. I'm working on new version of tabler-icons website 🙂

**Comment by OctopBP** [comment]
Looks like it's fixed 🙂
